flake.nix

```
@@ -5,16 +5,8 @@
 
   outputs = inputs:
     inputs.bellroy-nix-foss.lib.haskellProject {
-      cabalPackages = [
-        {
-          name = "botocore";
-          path = ./botocore.nix;
-        }
-      ];
-      supportedCompilers = [ "ghc8107" "ghc92" "ghc94" ];
+      src = ./.;
+      supportedCompilers = [ "ghc8107" "ghc92" "ghc912" "ghc94" ];
```

```suggestion
      supportedCompilers = [ "ghc8107" "ghc92" "ghc94" "ghc96" "ghc98" "ghc910" "ghc912" ];
```
Or the longest subsequence thereof that actually works.

---

Do I also need to put this in the cabal file  `tested-with`? Or is there some other way of knowing that it works?

---

Yes. `tested-with` drives CI, this only gives you `devShell` outputs in the flake.

---

Got it working with 3 versions.